integration_tests/mockito-kotlin/src/test/java/org/robolectric/integrationtests/mockito/kotlin/MockitoKotlinFunctionTest.java

```diff
@@ -0,0 +1,37 @@
+package org.robolectric.integrationtests.mockito.kotlin;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.*;
```

Comment:
I don't recommend all start imports, but it's okay to me.

Reply:
I will fix that.